go.mod

```diff
@@ -210,7 +210,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bgentry/speakeasy v0.1.0 // indirect
 	github.com/boombuler/barcode v1.0.1 // indirect
-	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
+	github.com/cenkalti/backoff/v3 v3.2.2 // indirect
```

Comment:
Wonder what brings this in as indirect?

Reply:
Ah:

```
# github.com/cenkalti/backoff/v3
github.com/openbao/openbao/command
github.com/openbao/openbao-template/config
github.com/openbao/openbao/api
github.com/cenkalti/backoff/v3
```

So we'll have to update `openbao-template` to use a new API tag after this is released in order to drop the `/v3` fully. That's fine!

Reply:
@JanMa Something to remember to do for v2.1.0!